sources/packages/web/src/views/institution/Login.vue

```diff
@@ -10,7 +10,7 @@
             Manage your institution account using your Business or Basic BCeID
             User ID. Learn how to setup and access your account
             <a
-              class="primary-color"
+              class="primary-color formio-href"
```

Comment:
`formio-href` has `color = $brand-blue;`. so we can remove `primary-color` class

Reply:
Thanks ann - good point!